.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+node_js:
+  - "stable"
+cache:
+  directories:
+    - "node_modules"
+install:
+  - npm install
```

Comment:
I think we decided to use yarn, or this has no effect on Travis?

Reply:
I don't think it has any effect on travis